docs/installing-solidity.rst

```diff
@@ -132,11 +132,7 @@ The nightly version can be installed using these commands:
 Furthermore, some Linux distributions provide their own packages. These packages are not directly
 maintained by us but usually kept up-to-date by the respective package maintainers.
 
-For example, Arch Linux has packages for the latest development version:
-
-.. code-block:: bash
-
-    pacman -S solidity
+For example, Arch Linux has packages for the latest development version as AUR packages: `solidity` and `solidity-bin`.
```

Comment:
Well, technically Arch does not *have* these packages. These are just build scripts maintained by random users and cannot be trusted to the same level as ones actually provided by the distro. People using AUR should know that, but still, this may be worth a note since just mentioning them like this sounds like we're recommending them.

To be fair these scripts look perfectly fine *right now* (`solidity` is based on the original official script and `solidity-bin` just installs our binary from a Github release), but they can be changed at any time without any kind of vetting from the distro.